locale/en/blog/release/v4.1.1.md

```diff
@@ -2,7 +2,7 @@
 date: 2015-09-23T02:01:38.545Z
 version: 4.1.1
 category: release
-title: Node v4.1.1 (Stable)
+title: Node v4.1.1 (Current)
```

Comment:
just seeing that this was discussed above. I for one would push towards not making a decision in this PR and supporting the old names. There is no way for us to update release commits, and I do not believe there is any intention of modifying the changelogs. It would be odd if the release posts + changelogs are not the same